features/support/manual_helpers.rb

```diff
@@ -333,4 +335,22 @@ def check_manual_is_withdrawn_from_panopticon(manual_slug)
         )
     end
   end
+
+  def get_id_for_manual(manual_title)
+    visit manuals_path
+    link = page.find_link(manual_title)
+    # TODO this is pretty gross, consider making it less so
+    link.native.attribute("href").value.match(%r{\A/manuals/(.*?)(\?|\z)})[1]
```

Comment:
We agree that this is pretty gross ([as before](https://github.com/alphagov/specialist-publisher/pull/450#discussion_r26955293)) but can't see another way of doing this right now. We think that showing the organisation's title on the manual page is a good idea, not just to make testing this easier but also to make the ownership clear to GDS editors, and to help with debugging potential issues where users think they should be able to see a manual but can't. Adding the organisation to the page is outside the scope of this PR, but this test can be simplified when we do that.